.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto
+
+*.sh text eol=lf
```

Comment:
We can drop that change.  I was suspecting the reason for an error I was getting was that even though I have LF only .sh files into my system it was coming out as CRLF on the test matrix.  

We need to make the change to enforced at some point.  But you are correct that  having it show full file conflicts is a problem right now.  I hate the "EOL hell" where files don't appear in a specified type each time it is pulled as it adds additional variables to testing.  The correct EOL settings require certain global settings appear on PC and MAC users machines and there is no way to guarantee that those settings are proper in the test matrix.  Thus long term we are better to fix all the files.